setup.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 # This file is protected via CODEOWNERS
 
+from __future__ import annotations
+
```

Comment:
We need to exclude `setup.py` because it needs to suport Python 2.7. This is why the `unsupported_setup_py` check is failing. Skipping the whole file seems to be the easiest way: https://pycqa.github.io/isort/docs/configuration/action_comments.html#isort-skip_file.

Reply:
Yep I'll skip the whole file